homework-submission/my-app/src/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import './App.css';
+import TodoItems from './Components/createDynamicItems';
+import Button from './Components/button';
+
+class Todolist extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [
+        {
+          name: 'REACT Lesson',
+          deadline: 23 - 3 - 2019,
+          key: Date.now(),
+        },
+        {
+          name: 'Wach movie',
+          deadline: 23 - 3 - 2019,
+          key: Date.now() + 1,
+        },
+        {
+          name: 'Football',
+          deadline: 23 - 3 - 2019,
+          key: Date.now() + 2,
+        },
+      ],
+    };
+    this.addItem = this.addItem.bind(this);
+    this.deleteItem = this.deleteItem.bind(this);
+  }
+
+  addItem(e) {
+    if (this._inputElement.value !== '') {
```

Comment:
This is one of the basic use of an if statement.
But now we are an advanced people a bit, So for legibility what about:
```
// because these are applied to both case either a true condition or a false one
this._inputElement.value = ' ';
e.preventDefault();
// the condition goes here...
if (/* this._inputElement.value !== '' // as you have */) {
  // the case of a true condition
}
```